docs/apidocs/ibmq-provider.rst

```diff
@@ -1,4 +1,4 @@
-.. _qiskit-providers-ibmq:
+.. module:: qiskit
```

Comment:
The following error stemmed from `docs/apidocs/ibmq-provider.rst` sharing the same label as `docs/apidocs/ibmq_provider.rst`:

```
.../qiskit-ibmq-provider/qiskit/providers/ibmq/__init__.py:docstring of qiskit.providers.ibmq:1: WARNING: duplicate label qiskit-providers-ibmq, other instance in .../docs/apidocs/ibmq-provider.rst
```

As a result, I switched the label to `qiskit`, which is the label used for qiskit-terra and qiskit-aqua.